dev/archery/archery/integration/tester_go.py

```diff
@@ -26,7 +26,8 @@ class GoTester(Tester):
     CONSUMER = True
 
     # FIXME(sbinet): revisit for Go modules
-    GOPATH = os.getenv('GOPATH', '~/go')
+    HOME = os.getenv('HOME', '~')
```

Comment:
Why the change, just set `GOPATH` if you have something else?

Reply:
I don't know why `~` failed resolve to `$HOME` locally but this seemed like a way to save someone the next person from the frustration of reading the script to figure out that `GOPATH` needs set.